<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GAPT 1.0.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Res.qrc">
    <normaloff>:/icones/Icones/applications_system.png</normaloff>:/icones/Icones/applications_system.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QObject{
	font-family: Georgia, &quot;Times New Roman&quot;, Times, serif;
	font-size:15px;
    color:black;
}
QMenuBar, QMenu , QMenuBar::item{
	background-color: #CCCCCC;
	color:black;
}

QMenuBar::item:selected, QMenu::item:selected{
	background-color:#363636;
	color:white;
}

QToolBar{
	background-color:#CCCCCC;
	spacing: 3px;
    border:0px;
}

QStatusBar, QSplitter::handle{
	background-color:#CCCCCC;
	color:white;
}
QScrollBar::handle:horizontal{
	border: 1px solid dark;
	border-radius:3px;
	background:#363636;
	height:15px;
	margin:0px 20px 0 20px;
}

QScrollBar::handle:horizontal{
	background:#777777;
	min-width: 20px;
}

QScrollBar::handle:horizontal:hover{
	background:#202020;
}

QScrollBar::add-line:horizontal{
	background:#777777;
	width:16px;
	subcontrol-position: right;
	subcontrol-origin: margin;
	border: 1px solid black;
	position: absolute;
	right:20px;
}

QScrollBar:left-arrow:horizontal, QScrollBar::right-arrow:horizontal{
	width:3px;
	height:3px;
	background:gray;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal{
	background: none;
}

QScrollBar:vertical{
	border: 1px solid black;
	border-radius:3px;
	background:#363636;
	width: 15px;
	margin: 20px 0 20px 0;
}

QScrollBar::handle:vertical{
	background:#777777;
	min-height:20px;
}

QScrollBar::add-line:vertical{
	border:1px solid black;
	background:#777777;
	height:16px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
	border: 1px solid black;
	background:#181818;
	height: 16px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical , QScrollBar::down-arrow:vertical{
	width: 3px;
	height: 3px;
	background: gray;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}

QTabWidget::pane {
	border: 1px solid black;
}

QTabBar::tab:selected  , QTabBar::tab:hover{
	background:qlineargradient(x1:0 , y1:0, x2:0 , y2: 1,
									stop: 0 #555555,
									stop: 1.0 #363636);
}

QTabBar::tab:selected {
	margin-top: 2px;
}

QDialog {
	background-color: #CCCCCC;
}
QDialog&gt;QlineEdit
{
	background-color:#FFFFFF;
    color:black;
}

/*QPushButton {
	color: black;
	border: 2px solid black;
	padding: 5px;
	min-width: 50px;
	margin: 3px;
	border-radius:3px;
}*/

QPushButton:hover{
	background-color:qlineargradient(x1:0 , y1:0, x2:0 , y2: 1);
}

QPushButton:pressed {
	background-color:qlineargradient(x1:0 , y1:0, x2:0 , y2: 1);
}

QLineEdit {
	background-color:#FFFFFF;
	color:black;
	selection-color:#363636;
	selection-background-color:#363636;
	border: 1px solid gray;
	border-radius: 3px;
}
QLineEdit::hover , QDateEdit:hover, QComboBox:hover {
		background-color:qlineargradient(x1:0 , y1:0, x2:0 , y2: 1);
		color:white;
}

 QLabel , QToolTip{
	background-color: #CCCCCC;
	color:black;
}
QFrame{
	background-color: #CCCCCC;
	color:black;
}

QHeaderView::section {
	background-color:qlineargradient(x1:0 , y1:0, x2:0 , y2: 1,
									stop: 0 #555555,
									stop: 1.0 #363636);
	color:white;
	border:1px solid black;
}

QHeaderView::section:hover{
	background-color:qlineargradient(x1:0 , y1:0, x2:0 , y2: 1, 
						            stop: 0 #555555,
									stop: 1.0 #363636);
	color:white;
}

QListeView::item::selected:active , QTreeView::item:selected:active {
	background-color:qlineargradient(x1:0 , y1:0, x2:0 , y2: 1,
										stop:0 #424242, stop:1 #2d2d2d);
	border: 1px solid darkgray;
	border-radius: 3px;
}

QListeView::item:hover, QTreeView::item:hover {
	background-color:qlineargradient(x1:0 , y1:0, x2:0 , y2: 1,
										stop:0 #2f2f2f, stop:1 #1f1f1f);
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelMenu">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:72pt; font-weight:600; color:#55aaff;&quot;&gt;GAPT 1.0.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGAPT">
    <property name="title">
     <string>GAPT</string>
    </property>
    <addaction name="actionParametre"/>
    <addaction name="actionChangerPassword"/>
    <addaction name="actionUtilisateur"/>
    <addaction name="actionCorbeil"/>
    <addaction name="actionDeconnexion"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuPersonel">
    <property name="title">
     <string>Personnel</string>
    </property>
    <addaction name="actionConge"/>
    <addaction name="actionPermission"/>
    <addaction name="actionDepart"/>
    <addaction name="actionSanction"/>
    <addaction name="actionFormation"/>
    <addaction name="actionCategorie"/>
   </widget>
   <widget class="QMenu" name="menuTransit">
    <property name="title">
     <string>Transit</string>
    </property>
    <addaction name="actionTransporteur"/>
    <addaction name="actionClient"/>
    <addaction name="actionDepense"/>
    <addaction name="actionReglement"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuGAPT"/>
   <addaction name="menuPersonel"/>
   <addaction name="menuTransit"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionParametre"/>
   <addaction name="actionChangerPassword"/>
   <addaction name="actionUtilisateur"/>
   <addaction name="actionCorbeil"/>
   <addaction name="actionDeconnexion"/>
   <addaction name="separator"/>
   <addaction name="actionConge"/>
   <addaction name="actionPermission"/>
   <addaction name="actionDepart"/>
   <addaction name="actionSanction"/>
   <addaction name="actionFormation"/>
   <addaction name="actionCategorie"/>
   <addaction name="separator"/>
   <addaction name="actionTransporteur"/>
   <addaction name="actionClient"/>
   <addaction name="actionDepense"/>
   <addaction name="actionReglement"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Georgia,Times New Roman,Times,serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>65</width>
     <height>65</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAgent"/>
   <addaction name="actionDepartement"/>
   <addaction name="separator"/>
   <addaction name="actionDossier"/>
   <addaction name="actionTransport"/>
   <addaction name="separator"/>
   <addaction name="actionQuitter"/>
  </widget>
  <action name="actionDepartement">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/batiment-medecine-pharmacie-image-png.png</normaloff>:/icones/Icones/batiment-medecine-pharmacie-image-png.png</iconset>
   </property>
   <property name="text">
    <string>Departement</string>
   </property>
   <property name="iconText">
    <string>      DÃ©partement     </string>
   </property>
  </action>
  <action name="actionAgent">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/perso5.jpg</normaloff>:/icones/Icones/perso5.jpg</iconset>
   </property>
   <property name="text">
    <string>Agent</string>
   </property>
   <property name="iconText">
    <string>          Agent            </string>
   </property>
   <property name="toolTip">
    <string>Agent</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDossier">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/doss4.jpg</normaloff>:/icones/Icones/doss4.jpg</iconset>
   </property>
   <property name="text">
    <string>Dossier     </string>
   </property>
   <property name="iconText">
    <string>         Dossier           </string>
   </property>
  </action>
  <action name="actionTransport">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/transport-camion-vehicule-jaune-icone-4890-128.png</normaloff>:/icones/Icones/transport-camion-vehicule-jaune-icone-4890-128.png</iconset>
   </property>
   <property name="text">
    <string>Transport</string>
   </property>
   <property name="iconText">
    <string>       Transport        </string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/quit3.jpg</normaloff>:/icones/Icones/quit3.jpg</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="iconText">
    <string>     Quitter     </string>
   </property>
  </action>
  <action name="actionParametre">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/settings_64.png</normaloff>:/icones/Icones/settings_64.png</iconset>
   </property>
   <property name="text">
    <string>Parametre</string>
   </property>
  </action>
  <action name="actionChangerPassword">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/mp1.jpg</normaloff>:/icones/Icones/mp1.jpg</iconset>
   </property>
   <property name="text">
    <string>ChangerPassword</string>
   </property>
  </action>
  <action name="actionDeconnexion">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/lock_64.png</normaloff>:/icones/Icones/lock_64.png</iconset>
   </property>
   <property name="text">
    <string>Deconnexion</string>
   </property>
  </action>
  <action name="actionUtilisateur">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/perso4.jpg</normaloff>:/icones/Icones/perso4.jpg</iconset>
   </property>
   <property name="text">
    <string>Utilisateur</string>
   </property>
  </action>
  <action name="actionConge">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/imgres.jpg</normaloff>:/icones/Icones/imgres.jpg</iconset>
   </property>
   <property name="text">
    <string>Conge</string>
   </property>
  </action>
  <action name="actionPermission">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/29-House-icon.png</normaloff>:/icones/Icones/29-House-icon.png</iconset>
   </property>
   <property name="text">
    <string>Permission</string>
   </property>
  </action>
  <action name="actionDepart">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/depart.jpg</normaloff>:/icones/Icones/depart.jpg</iconset>
   </property>
   <property name="text">
    <string>Depart</string>
   </property>
  </action>
  <action name="actionSanction">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/sanc.jpg</normaloff>:/icones/Icones/sanc.jpg</iconset>
   </property>
   <property name="text">
    <string>Sanction</string>
   </property>
  </action>
  <action name="actionFormation">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/form.jpg</normaloff>:/icones/Icones/form.jpg</iconset>
   </property>
   <property name="text">
    <string>Formation</string>
   </property>
  </action>
  <action name="actionCategorie">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/application_vnd_oasis_opendocument_database.png</normaloff>:/icones/Icones/application_vnd_oasis_opendocument_database.png</iconset>
   </property>
   <property name="text">
    <string>Categorie</string>
   </property>
  </action>
  <action name="actionTransporteur">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/trans.jpg</normaloff>:/icones/Icones/trans.jpg</iconset>
   </property>
   <property name="text">
    <string>Transporteur</string>
   </property>
  </action>
  <action name="actionClient">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/perso2.jpg</normaloff>:/icones/Icones/perso2.jpg</iconset>
   </property>
   <property name="text">
    <string>Client</string>
   </property>
  </action>
  <action name="actionDepense">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/gnome_mime_application_qif.png</normaloff>:/icones/Icones/gnome_mime_application_qif.png</iconset>
   </property>
   <property name="text">
    <string>Depense</string>
   </property>
  </action>
  <action name="actionReglement">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/imgres3.jpg</normaloff>:/icones/Icones/imgres3.jpg</iconset>
   </property>
   <property name="text">
    <string>Reglement</string>
   </property>
  </action>
  <action name="actionCorbeil">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/icones/Icones/02-Recycle-icon.png</normaloff>:/icones/Icones/02-Recycle-icon.png</iconset>
   </property>
   <property name="text">
    <string>Corbeille</string>
   </property>
   <property name="toolTip">
    <string>Corbeille</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Res.qrc"/>
 </resources>
 <connections/>
</ui>
